extra/migrations/27032015000000_migrate_monolithic_config.rb

```diff
@@ -0,0 +1,112 @@
+#
```

Comment:
The filename here is perhaps the wrong way around (DDMMYYYY) if we're sorting the filenames, should it be YYYYMMDD?